spark-common/src/main/js/src/spark-common.soy

```diff
@@ -90,3 +95,31 @@
         </div>
     </div>
 {/template}
+
+/**
+ * @param id Id of the HTML element
+ * @param src URL of the content to load to the iframe
+ * @param createOptions extra options to customize the dialog
+ */
+{template .appFullscreenContaineriFrame}
+    <div id="{$id}" class="spark-fullscreen-wrapper">
+        {if $createOptions.addChrome}
+        <div class="spark-fullscreen-chrome">
+            <div class="spark-fullscreen-chrome-btnwrap">
+                <button id="{$id}-chrome-submit"
+                    class="aui-button aui-icon aui-icon-small aui-iconfont-success">
+                    "Submit"
```

Comment:
Should read OK for now, instead of "Submit".

We'll need a way to define customizable buttons later on.